dnload/glsl_block_layout.py

```diff
@@ -36,7 +36,7 @@ def glsl_parse_layout(source):
         return (None, source)
     lst = []
     while scope:
-        (location, assignment, index, intermediate) = extract_tokens(scope, ("?|location", "?=", "?u"))
+        (location, assignment, index, intermediate) = extract_tokens(scope, ("?|location|binding", "?=", "?u"))
```

Comment:
```suggestion
        (location, assignment, index, intermediate) = extract_tokens(scope, ("?|binding|location", "?=", "?u"))
```
I just want these in alphabetic order because of autistic tendencies.